kola/tests/ignition/passwd.go

```diff
@@ -282,7 +282,7 @@ func usersRhcos(c cluster.TestCluster) {
 	tests := []userTest{
 		{
 			user:           "core",
-			passwdRecord:   "core:x:1000:1000:CoreOS Admin:/home/core:/bin/bash",
+			passwdRecord:   "core:x:1000:1000:CoreOS Admin:/var/home/core:/bin/bash",
```

Comment:
How about also:

```diff
diff --git a/kola/tests/ignition/passwd.go b/kola/tests/ignition/passwd.go
index c5cae9ce..e138e29e 100644
--- a/kola/tests/ignition/passwd.go
+++ b/kola/tests/ignition/passwd.go
@@ -219,7 +219,7 @@ func init() {
                               ]
                             }
                           }`),
-               Distros: []string{"rhcos"},
+               Distros: []string{"rhcos", "fcos"},
        })
 }

```

?

Since we should be able to run this as is on FCOS too now.

Reply:
Right...also just noting here as soon as we can drop CL this is another instance of tech debt that would just go away.